tests/drivers/dac/dac_loopback/prj.conf

```diff
@@ -0,0 +1,5 @@
+CONFIG_ZTEST=y
+
+CONFIG_DAC=y
+CONFIG_ADC=y
+CONFIG_TEST_USERSPACE=y
```

Comment:
I don't believe this test benefits from running under userspace. It should be enough that the basic API test runs in userspace (also, the test fails in userspace since you never grant access to neither the DAC nor the ADC device in `main.c`).